CefSharp.MinimalExample.Wpf/CefSharp.MinimalExample.Wpf.csproj

```diff
@@ -67,8 +67,10 @@
     <ApplicationManifest>app.manifest</ApplicationManifest>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.Expression.Interactions, Version=4.5.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
```

Comment:
They are currently using `Copy Local = True`, I can't see how this would be different since they come with VS, just not added in the projects by default.